server/server.go

```diff
@@ -1451,6 +1451,32 @@ func (s *Server) fetchAccount(name string) (*Account, error) {
 	return acc, nil
 }
 
+func (s *Server) enableOCSP() error {
+	opts := s.getOpts()
+	// Check if OCSP Stapling was enabled/disabled globally.
+	oc := opts.OCSPConfig
+	if oc == nil || oc.Mode == OCSPModeNever {
```

Comment:
If `OCSPModeAuto` is the default, then surely `oc == nil` should imply `OCSPModeAuto` and we should not return nil for that scenario?